test/helper/helper.go

```diff
@@ -670,8 +698,8 @@ func CleanTestVrfs(t *testing.T, vrfId string) {
 	t.Helper()
 	TestApiClient := TestClient()
 
-	resp, err := TestApiClient.VRFsApi.DeleteVrf(context.Background(), vrfId).Execute()
-	if err != nil && resp.StatusCode != http.StatusNotFound {
+	_, err := TestApiClient.VRFsApi.DeleteVrf(context.Background(), vrfId).Execute()
+	if err != nil && strings.Contains(err.Error(), "Not Found") {
```

Comment:
At a glance I would expect both the old and new code to work here, but the status code check is more explicit.  Could you change it back and we'll see if that fixes the test cleanup failure?

```
=== RUN   TestCli_Vrf_Create/vrf-delete-test
    helper.go:690: Error when calling `VRFsApi.DeleteVrf`` for 548a73da-78b0-484f-b7be-5c6d5e48e714: 404 Not Found Not found
```

Reply:
Yes reverted